mitodl/mitxonline
# Populate passing DEDP grades that were set to None by Admin

# What are the relevant tickets?
Related to https://github.com/mitodl/mitxonline/issues/1762

# Description (What does it do?)
Populate passing DEDP grades that were set to None by Admin
Question: Do we need to make sure these grades are verified?


# How can this be tested?
Have a user with a passing grade with letter_grade set to None. Then run migrations.
The grade should get populated with a letter grade.
